cmd/crank/beta/render/cmd.go

```diff
@@ -84,10 +84,14 @@ the following annotations to each Function to change how they're run:
 
     Don't stop the Function's Docker container after rendering.
 
+  render.crossplane.io/runtime-docker-name: "<name>"
```

Comment:
It's a bit long, but maybe `runtime-docker-container-name` would make it super clear?

Reply:
i did not want to changed the default behavior also i wanted to have flexibility and not just name the container like the image.

also i think runtime-docker-name also enough also in the scheme of runtime-docker-image, <imho>

Reply:
we could add if no name is given use the image name as a default name instead of empty one. But it could also break thing, e.g. you do that one some system where you have also the named once which would possible lead to them beeing deleted. which is not what someone might want.

also i will probably update my test script so in the last render it just uses another function.yaml which deletes the container after for cleanup.